documentation/general/decouple-vs-and-net-sdk.md

```diff
@@ -170,6 +177,9 @@ There is only one version of the DevKit extension. It is released using the late
 - [Long term build-server shutdown issue](https://github.com/dotnet/msbuild/issues/10035)
 - [MSBuild decouple issue](https://github.com/dotnet/msbuild/issues/11142)
 
+[^1]: This specific problem is now largely mitigated in `msbuild` scenarios by the .NET SDK implicitly adding the Microsoft.Net.Sdk.Compilers.Toolset package to builds in a torn state. This will install and use a newer C# compiler when MSBuild is older than the .NET SDK. That fixes a large amount of the problems but has a number of downsides: increases restore time, compiler must go through full JIT, corner case scenarios like `$(NUGET_PACKAGES)` not being set lead to errors, only fixes one type of torn state, etc ... 
```

Comment:
```suggestion
[^1]: This specific problem is now largely mitigated in `msbuild` scenarios by the .NET SDK implicitly adding the Microsoft.Net.Sdk.Compilers.Toolset package to builds in a torn state. This will install and use a newer C# compiler when MSBuild is older than the .NET SDK. That fixes a large amount of the problems but has a number of downsides: increases restore time, compiler must go through full JIT, corner case scenarios like `$(NuGetPackageRoot)` not being set lead to errors, only fixes one type of torn state, etc ... 
```